dotnet-monitor.yml

```
@@ -9,11 +9,25 @@ pr:
     - internal/release/*
     - feature/*
 
+schedules:
+- cron: 0 12 * * 6
```

```suggestion
- cron: 15 11 * * 6
```

I've had issues with everyone kicking off scheduled jobs at regular hours like "noon Saturday" or "midnight Sunday", this would cause build pools would get overwhelmed. Don't really know if that would happen here but might be worth it to pin the startup time to something weirder.